composer.json

```diff
@@ -8,12 +8,15 @@
         "bacon/bacon-qr-code": "^2.0",
         "calebporzio/parental": "^0.11.0",
         "codezero/laravel-unique-translation": "^3.0",
-        "doctrine/dbal": "^2.9",
+        "doctrine/dbal": "^2.10",
+        "facade/ignition": "^2.0",
         "fideloper/proxy": "^4.1",
         "jenssegers/date": "^4.0",
-        "laravel/framework": "^6.0",
-        "laravel/tinker": "^2.0",
+        "laravel/framework": "^7.0",
+        "laravel/tinker": "^2.4",
+        "laravel/ui": "^2.0",
```

Comment:
Do we need this? It looks like an auth UI scaffolding library.

Reply:
Turns out Laravel moved some stuff from core we're depending on into this package, so it is required.

https://github.com/laravel/ui/tree/3.x/auth-backend